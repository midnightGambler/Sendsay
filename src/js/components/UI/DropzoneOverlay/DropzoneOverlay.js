import React from "react";

const DropzoneOverlay = () => (
  <div className="dropzone-overlay">
    <h3 className="dropzone-overlay__title">Бросайте файлы сюда, я ловлю</h3>
    <p className="dropzone-overlay__subtitle">
      Мы принимаем картинки (jpg, png, gif), офисные файлы (doc, xls, pdf) и
      zip-архивы.
      <br />
      Размеры файла до 5 МБ
    </p>
  </div>
);

export default DropzoneOverlay;
