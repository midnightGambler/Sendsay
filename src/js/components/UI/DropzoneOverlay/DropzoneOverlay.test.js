import React from "react";
import DropzoneOverlay from "./DropzoneOverlay";
import { shallow } from "enzyme";

it("should render component properly", () => {
  const dropzoneOverlay = shallow(<DropzoneOverlay />);
  expect(dropzoneOverlay).toMatchSnapshot();
});
