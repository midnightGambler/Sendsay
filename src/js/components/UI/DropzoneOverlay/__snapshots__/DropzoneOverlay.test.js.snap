// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component properly 1`] = `
<div
  className="dropzone-overlay"
>
  <h3
    className="dropzone-overlay__title"
  >
    Бросайте файлы сюда, я ловлю
  </h3>
  <p
    className="dropzone-overlay__subtitle"
  >
    Мы принимаем картинки (jpg, png, gif), офисные файлы (doc, xls, pdf) и zip-архивы.
    <br />
    Размеры файла до 5 МБ
  </p>
</div>
`;
